tests/unit/handlers/unit_tests_receiver_handler.rb

```diff
@@ -32,6 +32,7 @@ def setup
     @idle_timeout_value = 85
     @max_frame_size_value = 8191
     @log_lib_value = "TRANSPORT_RAW"
+    @auto_settle_off_value = "settle"
```

Comment:
This is invalid value for the parameter, it will be parsed from cli as a true/false. I think you should use a valid nondefalut value here, which is `true`.

Reply:
Yes, it is invalid value. This is testing that constructor parameter is set and used. I can use `not Defaults::...` to set it to non-default value, because nobody will care about changed default value in relation to unit tests.